src/components/Checkbox/index.scss

```diff
@@ -1,48 +1,48 @@
-@use '~styles/colors.scss' as colors;
-@use '~styles/fonts.scss' as fonts;
-
-.checkbox-container {
-  display: inline-flex;
-
-  align-items: center;
-
-  margin: 1rem;
-
-  cursor: pointer;
-
-  &__input {
-    width: 2.2rem;
-    height: 2.2rem;
-
-    margin: 0.8rem;
-    border: 0.2rem solid colors.$gray500;
-
-    border-radius: 0.5rem;
-
-    cursor: pointer;
-    appearance: none;
-
-    &:checked {
-      border-color: colors.$primary500;
-
-      background: colors.$primary500 url('./images/check.svg') no-repeat center
-        center/70%;
-    }
-
-    &:disabled {
-      border-color: colors.$gray500;
-
-      cursor: not-allowed;
-    }
-  }
-
-  &__checkboxText {
-    font-family: fonts.$primaryFont;
-    color: colors.$gray800;
-    text-align: left;
-    font-size: fonts.$xs;
-    font-weight: fonts.$semiBold;
-    line-height: 2;
-    letter-spacing: 0;
-  }
-}
+@use '~styles/colors.scss' as colors;
+@use '~styles/fonts.scss' as fonts;
+
+.checkbox-container {
+  display: inline-flex;
+
+  align-items: center;
+
+  margin: 1rem;
+
+  cursor: pointer;
+
+  &__input {
+    width: 2.2rem;
+    height: 2.2rem;
+
+    margin: 0.8rem;
+    border: 0.2rem solid #ACACB5; //cor do Figma não está na paleta de cores do projeto
```

Comment:
se eu não me engano agora tem um arquivo de cor no projeto, adiciona a cor lá e usa a variável aqui e pode remover o comentário também por favor

Reply:
Ale orientou a conversar com o designer sobre isso, eu mandei msg pra ele agora, to esperando a resposta

Reply:
Não consegui resposta do designer, eu adicionei a cor $gray250: rgb(172, 172, 181); no arquivo colors.scss